vulcanai/models/basenetwork.py

```diff
@@ -877,53 +878,43 @@ def run_test(self, data_loader, plot=False, save_path=None, pos_label=1,
     def bootfold_p_estimate(self, data_loader, n_samples, k, epochs, 
                             index_to_iter, ls_feat_vals, retain_graph=None,
                             valid_interv=4, plot=False, save_path=None,
-                            transform_outputs=False, transform_callable=None, p_output_path = None,
```

Comment:
is the transform_callable parameter deprecated?

Reply:
no yet, but it's not used in this function as Joe implemented it, so I wanted to take it out to avoid thoughts that it was used. I want to get rid of it anyways, in a future pull request.